ouroboros-network/test/Test/Ouroboros/Network/PeerSelection.hs

```diff
@@ -1909,6 +1910,7 @@ _governorFindingPublicRoots targetNumberOfRootPeers readDomains =
 
         peerSelectionGovernor
           tracer tracer tracer
+          (mkStdGen 42)
```

Comment:
Shouldn't this constant (and the other one above) be generated by quickcheck?

Reply:
I'm unsure of the ramifications of taking the seed from quickcheck.

Reply:
To cover more possible interleavings of events.

Reply:
The governor test suite is too brittle to handle that. Sorting that out should be done in Prong 2, not in this PR.

Reply:
Please add a TODO, and create an issue for this then.

Reply:
Created #3182